### ISSUE TITLE
var=$( sed 's/a/b/' <<< "$var" ) doesn't trigger sc2001

### ISSUE THREAD

**Issue by izabera** [koalaman/shellcheck#770]
what the title says

**Comment by Scorpiokat** [comment]
Indeed from the Wiki description of SC2001 it sounds like it should've been triggered. But from the code behind SC2001 (see snippet below), it would appear use of `echo` is expected just like in the [example in SC2001](https://github.com/koalaman/shellcheck/wiki/SC2001).
```haskell
prop_checkEchoSed1 = verify checkEchoSed "FOO=$(echo \"$cow\" | sed 's/foo/bar/g')"
prop_checkEchoSed2 = verify checkEchoSed "rm $(echo $cow | sed -e 's,foo,bar,')"
checkEchoSed = ForShell [Bash, Ksh] f
  where
    f (T_Pipeline id _ [a, b]) =
        when (acmd == ["echo", "${VAR}"]) $
            case bcmd of
                ["sed", v] -> checkIn v
                ["sed", "-e", v] -> checkIn v
                _ -> return ()
```
Code is from around line 307 (as of today) in [ShellSupport.hs](https://github.com/koalaman/shellcheck/blob/master/ShellCheck/Checks/ShellSupport.hs).

**Comment by Gandalf-** [comment]
This is a duplicate of https://github.com/koalaman/shellcheck/issues/130, which has been fixed

**Comment by koalaman** [comment]
Indeed it has. Thanks!
